index.html

```diff
@@ -40,5 +40,209 @@ <h1>I'm Benjamin Glad to see you!</h1>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="text-area">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+      </div>
+
+      <div class="middle-section-nav-bars">
+        <ul class="technology">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+      </div>
+
+      <div class="project-link">
+        <h1>See Project</h1>
+      </div>
+  </section>
+
+  
+  <section class="middle-section-container">
+    <div class="image-area">
+      <img src="photo/logos/Snapshoot-Portfolio3.png" alt="">
+    </div>
+
+  <div class="hero">
+      <div class="tonic">
+          <h1>Tonic</h1>
+        <div class="canopy-wrapper">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="text-area">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+      </div>
+    </div>
+
+    <div class="middle-section-nav-bars">
+      <ul class="technology">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+    </div>
+
+    <div class="project-link">
+      <h1>See Project</h1>
+    </div>
+</section>
+
+
+<section class="middle-section-container">
+  <div class="image-area">
+    <img src="photo/logos/Snapshoot-Portfolio4.png" alt="">
+  </div>
+
+<div class="hero">
+    <div class="tonic">
+        <h1>Tonic</h1>
+      <div class="canopy-wrapper">
+        <p>CANOPY</p>
+        <ul>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="text-area">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+    </div>
+  </div>
+
+  <div class="middle-section-nav-bars">
+    <ul class="technology">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+  </div>
+
+  <div class="project-link">
+    <h1>See Project</h1>
+  </div>
+</section>
+
+<section class="about-myself">
+  <h1 class="header">
+    About Myself
+  </h1>
+
+  <div class="header-text">
+    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  </div>
+
+  <div class="about-myself-link">
+    <p>LET'S CONNECT</p>
+  </div>
+
+  <div class="about-myself-icons">
+    <img src="photo/logos/github.jpg" alt="">
+    <img src="photo/navs/Linkedin-icon.jpg" alt="">
+    <img src="photo/logos/angellist.jpg" alt="">
+    <img src="photo/logos/twitter.jpg" alt="">
+    <img src="photo/logos/medium.jpg" alt="">
+  </div>
+
+  <div class="resume">
+    <p>Get my resume</p>
```

Comment:
- [ ] As I can see here that you didn't add  the button tag so kindly put `button` instead of `p` because here  this is a button so you get the resume and see it, and add styling: Active, Hover, and Pressed please follow the styling and add them to your button to match the Figma

- [ ] I think it would be better if you wrap the button in `a` tag because in the future you will need to put a real project so the recruiters can see your project well 👌🏻👌🏻💯